zetaclient/chains/evm/signer/signer.go

```diff
@@ -153,6 +157,12 @@ func (signer *Signer) GetERC20CustodyAddress() ethcommon.Address {
 	return signer.er20CustodyAddress
 }
 
+// GetGatewayAddress returns the gateway address
+func (signer *Signer) GetGatewayAddress() string {
+	// return empty string for now
+	return ""
```

Comment:
please add TODOs for those Set / Get GatewayAddress calls

Reply:
I'll add a comment for clarification. The gateway doesn't exist for EVM chain now, it will be available until we switch to the new contract architecture.

Reply:
https://github.com/zeta-chain/node/pull/2560/commits/e88593e01650473035620204a07edaac44965157